keyboards/handwired/m40/5x5_macropad/info.json

```
@@ -0,0 +1,41 @@
+{
+    "keyboard_name": "5x5_macropad", 
+    "url": "https://www.thingiverse.com/thing:5239739", 
+    "maintainer": "m40-dev", 
+    "layouts": {
+        "LAYOUT": {
```

QMK Configurator won't render the board layout properly unless this is changed.

```suggestion
        "LAYOUT_ortho_5x5": {
```